src/main/content/_layouts/post.html

```diff
@@ -1,49 +1,67 @@
+                <div class="share_post_container">
+                    <a class="twitter_link" href="//twitter.com/intent" onclick="window.open('https://twitter.com/intent/tweet?text=' + encodeURIComponent('{{page.title}}') + '&url=' + encodeURIComponent(window.location)); return false" aria-label="Twitter link"></a>
+                    <a class="reddit_link" href="//www.reddit.com/submit" onclick="window.open('//www.reddit.com/submit?url=' + encodeURIComponent(window.location)); return false" aria-label="Reddit link"></a>
                 </div>
+                <h1 id="post_title">{{ page.title | escape }}</h1>
+                <div id="post_information_container">
+                    {% assign additional_author_count = 0 %}
+                    {% for author in page.additional_authors %}
+                        {% assign additional_author_count = additional_author_count | plus: 1 %}
+                    {% endfor %}
+                    {% assign current = 1 %}
+                    <div class="author_images_container">
+                        <img src="{{ page.author_picture }}" class="author_image" alt="image of author">
+                        {% for author in page.additional_authors %}
+                            <img src="{{ author.image }}" class="author_image" alt="image of author">
+                        {% endfor %}
+                    </div>
+                    <a class="author_name" href="{{ page.author_github }}" target="_blank" rel="noopener">{{ page.author }}</a>
+                    {% for author in page.additional_authors %}
+                        {% if additional_author_count == 1 %}
+                            <span>and</span>
+                        {% elsif additional_author_count > 1 %}
+                            {% if current < additional_author_count %}
+                                <span class="comma">,</span>
+                            {% elsif current == additional_author_count %}
+                                <span class="comma">, and</span>
+                            {% endif %}
+                        {% endif %}
+                        {% assign current = current | plus: 1 %}
+                        <a class="author_name" href="{{ author.github }}" target="_blank" rel="noopener">{{ author.name }} </a>
+                    {% endfor %}
+                    {% assign date_format = site.minima.date_format | default: "%b %-d, %Y" %}
+                    <span id="post_date">on {{ page.date | date: date_format }}</span>
+                </div>
+            </div>
+        </header>
+        <div id="article_body" class="container">
+            {{ content }}
+            <div class="share_post_container">
+                <a class="twitter_link" href="//twitter.com/intent" onclick="window.open('https://twitter.com/intent/tweet?text=' + encodeURIComponent('{{page.title}}') + '&url=' + encodeURIComponent(window.location)); return false" aria-label="Twitter link"></a>
```

Comment:
Same thing as above

Reply:
will this be taken care when Ellen do the merge?